parrot.el

```diff
@@ -169,15 +169,17 @@ For example, an animation with a total of ten frames would have a
         ((string= parrot "thumbsup") 12)
         (t (error (format "Invalid parrot %s" parrot)))))
 
-(defun parrot-set-parrot-type (parrot)
+(defun parrot-set-parrot-type (parrot &optional silent)
   "Set the desired PARROT type in the mode line."
   (interactive (list (completing-read "Select parrot: "
                                       '(default confused emacs nyan rotating science thumbsup) nil t)))
   (setq parrot-frame-list (number-sequence 1 (parrot-sequence-length parrot)))
-      (setq parrot-type parrot)
-      (parrot-load-frames parrot)
+  (setq parrot-type parrot)
+  (parrot-load-frames parrot)
+  (unless silent
+    (progn
```

Comment:
Thanks for the PR! Just one quick note; `progn` is not needed for `unless` conditionals. Please remove it and I'll go ahead and get this merged.